changes/6354.added

```diff
@@ -0,0 +1 @@
+Bulk edit operation now a system job.
```

Comment:
```suggestion
Added "Bulk Edit Objects" system job.
```

I'd suggest also adding a .changed fragment for the change in behavior of the UI.